### DISCUSSION TITLE
How to support the latest ChatGPT model (1106)?

### DISCUSSION THREAD

**Discussion by yiwaixiao** [Zain-ul-din/whatsapp-ai-bot#18] [Ideas]
How to support the latest ChatGPT model (1106)?

**Comment by Zain-ul-din** [comment]
### To change the default model 

- go to source code `src/whatsapp-ai.config.ts`
- set `chatGPTModel: "gpt-3.5-turbo"` to `chatGPTModel: "gpt-4-32k"` **OR** add a model of your choice inside `chatGPTModel: "mode_name"`

![image](https://github.com/Zain-ul-din/whatsapp-ai-bot/assets/78583049/06364356-6ca9-4281-bc2a-c9136b4bbcff)



![image](https://github.com/Zain-ul-din/whatsapp-ai-bot/assets/78583049/2346b628-3106-4149-8999-7287431e2f52)

### check out more [here](https://platform.openai.com/docs/models/gpt-4)
